src/thread.js

```diff
@@ -0,0 +1,107 @@
+class Thread {
+  /**
+   * Please use **space.joinThread** to get the instance of this class
+   */
+  constructor (orbitdb, name, threeId, subscribe, ensureConnected) {
+    this._orbitdb = orbitdb
+    this._name = name
+    this._3id = threeId
+    this._subscribe = subscribe
+    this._ensureConnected = ensureConnected
+    this._queuedNewPosts = []
+  }
+
+  /**
+   * Post a message to the thread
+   *
+   * @param     {Object}    message                 The message
+   * @return    {String}                            The postId of the new post
+   */
+  async post (message) {
+    this._requireLoad()
+    this._subscribe()
+    this._ensureConnected(this._address, true)
+    return this._db.add({
+      author: this._3id.getDid(),
+      message,
```

Comment:
do we want any max length vals?

Reply:
That might be a good idea! Do you know of a good way to implement that?

Reply:
that's interesting, I was thinking about that the other day. Maybe something large - i think Disqus uses 20,000 which is like 6 pages single spaces on MSFT word

Reply:
We are not enforcing the use of strings though :)

Reply:
just string length, but actually doesn't matter in this first version, wait to till later and we havent done the same for other vals yet

Reply:
My point is that we don't enforce it to be a string anywhere, so it might be an object, an array, or anything.

Reply:
right, but doesn't orbit eventually cast to string? or bytes? would cast the same as orbit and do length

Reply:
and I guess maybe we just want strings for threads? not sure

Reply:
I think we want mixed media threads, so not just "strings" but I imagine as Zach said, Orbit boils everything down to strings in the end, right?

Reply:
I'm not actually sure about the internals of orbitdb here.